website/docs/explanation/glossary_what.md

```diff
@@ -0,0 +1,129 @@
+
+```{glossary}
+core
+ The working memory of a computer.
+```
+
+### Abbreviation
+
+If the term is an abbreviation, the gloss starts with its expansion. The abbreviation is explained briefly after its expansion.
+
+```{glossary}
+DNS
+ **Domain Name System**. A system that translates human-readable domain names (canonical.com) to their IP addresses (185.125.190.20).
+```
+
+In a conversational style, the definition has more of the structure and tone of a spoken sentence.
+For example, the definition may begin by repeating the term being defined.
+
+```{glossary}
+FC
+ **Fiber Channel** is a storage networking protocol used for low-latency communication between a storage device and a node
+ in a Storage Area Network (SAN).
+```
+
+In this definition of FC, the abbreviation SAN is preceded by its expansion as is customary in running text.
+This is a good way to handle abbreviations that may be unfamiliar or have not yet been formally defined.
+
+### Cross-references
+
+Cross-references help readers navigate among related terms.
+
+If the gloss for a term discusses a second term, the second term should be linked.
+
+<!-- {term}`DNS` replaced by *DNS* -->
+<!-- other terms: LDAP, SSL, TLS, DTLS -->
+
+
+```{glossary}
+DDNS
+ **Domain Name System**.
+ A service that automatically updates :term:`DNS` records when the underlying IP address changes (aka, dynamic IP).
+```
+
+When a concept that provides depth or broader context is used in a gloss, it can be linked using "See".
+
+```{glossary}
+DIT
+ **Directory Information Tree**. In directory services (See :term:`LDAP`), a hierarchical tree-like structure used to organize and store information.
+```
+
+A collection of related topics can be provided at the end of the gloss.
+
+```{glossary}
+GnuTLS
+ **GNU’s Not Unix Transport Layer Security**.
+ A GNU software package that secures data in transit by implementing the :term:`SSL`, :term:`TLS`, and :term:`DTLS` protocols.
+
+ Related topic(s): [GnuTLS (Ubuntu Server documentation)](https://documentation.ubuntu.com/server/explanation/crypto/gnutls/#), [GnuTLS (official site)](https://www.gnutls.org/), Cryptography, Web services, OpenLDAP.
+
+```
+
+## Advanced features
+
+In this treatment, the features considered basic are those focused on meaning.
+
+There is a further realm of glossary features that are focused on terminology management. Sometimes the origin of a term, or the pronunciation, or an alternate form, or some other aspect of the term is of great interest.
+
+Advanced features can be introduced in glossaries by imitating the features of entries in authoritative dictionaries.
+
+## Appendix 1: Mechanics
+
+This document is written in the following hierarchy of languages (high to low):
+* Markedly Structured Text ([MyST](https://myst-parser.readthedocs.io/en/latest/syntax/typography.html#syntax-glossaries))
+* reStructured Text (rST)
+* Markdown (md)
+
+A reference to a term is coded as follows: ::
+
+  :term:`referenced_term`
```

Comment:
```suggestion
`{term}`referenced_term``
```

Reply:
I'm not sure if this will render as you intended, but have a look at if not I'll tweak it again